File: BarCodeReaderDemoAPP/app/src/main/java/com/example/rohit/barcodereaderdemoapp/MainActivity.java
package com.example.rohit.barcodereaderdemoapp;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity implements BarcodeReader.BarcodeReaderListener
{

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: CommonToolbar/app/src/main/java/com/example/rohit/commontoolbar/MainActivity.kt
package com.example.rohit.commontoolbar

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.Toolbar
import android.view.MenuItem
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity()
{


    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        toolbar.inflateMenu(R.menu.menu_popup);

    }


}
